python/query.py

```diff
@@ -90,5 +92,34 @@ def main():
         writer.writerow(columns)
 
 
+def parse_arguments():
+    """Parse command line arguments."""
+    parser = ArgumentParser(description="Query the Cantabular extended API using the GraphQL "
+                            "endpoint. Verify and parse the response and print the rows contained "
+                            "in the table in CSV format.")
+
+    # Positional arguments
+    parser.add_argument("base_url",
+                        metavar="URL",
+                        type=str,
+                        help="Cantabular GraphQL base URL including port "
+                             "e.g. http://localhost:8492/graphql")
+
+    parser.add_argument("dataset",
+                        metavar="Dataset",
+                        type=str,
+                        help="Name of dataset to use in query")
+
+    parser.add_argument("variables",
+                        metavar="Variables",
+                        nargs="+",
+                        type=str,
+                        help="Names of variables to use in query. At least one variable name "
+                        "must be supplied. A single rule variable may be specified. If a rule "
```

Comment:
You can supply at most one rule variable which must be first if present.
?

Reply:
OK